### ISSUE TITLE
COM4j not compatible with windows 64 bit machine

### ISSUE THREAD

**Issue by shanthia** [kohsuke/com4j#50]
I'm getting class not registered for com4j when i'm trying to access ALM from java using OTA connector. Its working fine with 32 bit machine. I've downloaded com4j from the below link which has dll for 64 bit machine. 1)http://kohsuke.org/2012/04/27/com4j-updates/ 2) https://github.com/kohsuke/com4j/downloads.
still i'm getting the same error. Is com4j compatible with windows 64 bit?

**Comment by shanthia** [comment]
Can anyone help me?

**Comment by kllbzz** [comment]
Hi, are you trying to use 64 bit java to connect to 32 bit native dll ? 
First I would play with bitness(32/64) of java and your native dll(com object), second would try to debug the whole stack java->com4j->you native dll to understand on what step is failing and google your error then

**Comment by luisbarros27** [comment]
I've the same problem Exception in thread "main" com4j.ExecutionException: com4j.ComException: 80040154 CoCreateInstance failed : Clase no registrada : .\com4j.cpp:153 
I can't run it in win64, for burocracy requirements I can't avoid java and windows64

**Comment by AnandChaurasia-zz** [comment]
I am also getting same error. I am using com4j to integrate with ALM

**Comment by kllbzz** [comment]
btw, did you see comment from kohsuke here https://java.net/jira/browse/COM4J-77?focusedCommentId=338656&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-338656

**Comment by usmcphysicist** [comment]
I have a unique problem, instead of writing it twice, I also posted it on StackOverflow [here](http://stackoverflow.com/questions/43727444/linkageerror-caused-by-classnotfoundexception-at-runtime-but-not-during-junit)
Any help would be appreciated.

**Comment by jacobbeasley** [comment]
Was this ever resolved?

**Comment by luisbarros27** [comment]
No, in my case, I had to change everything in my app, it was made in
delphi, and call it via activex, a disgusting crap solution.

**Comment by jacobbeasley** [comment]
Yeah, in my case I just used the 32-bit JVM and it seems to work now.

**Comment by CallSign-Filter** [comment]
It works sometimes...

If I register my dll with regsvr32 and pray to the gods, it works on some
systems but not on others. We will be moving away from this because of
inconsistencies and lack of support.
